python/pynessie/commands/contents/commit.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
```

Comment:
as everything has been moved the diff isn't terribly clear. Is this a re-write or a refactor of the old logic?

Reply:
There is _no_ new logic and it not a re-write. This is basically a refactor of the old logic and thus unit tests remained unchanged.